### DISCUSSION TITLE
Accesso impossibile con Home Assistant companion

### DISCUSSION THREAD

**Discussion by amadeus74** [indomus/forum#851] [HUB: Home Assistant]
Ciao a tutti.
Ho installato HASSIO sul mio Raspberry PI.
Accedevo con l'app Android da due diversi smartphone ma con uno di essi non riesco più ad accedere: nel momento in cui indico l'istanza di HA a cui connettersi e inserisco le credenziali, compare questo messaggio:
_Impossibile connettersi a Home Assistant,
Si è verificato un errore nel caricamento di Home Assistant, si prega di rivedere le impostazioni di connessione e riprovare.
Tentermo di provare un altro URL fornito quando cliccherai su Aggiorna._
E la situazione da qui non si sblocca.
Ho disinstallato e reinstallato l'app più volte ma non cambia nulla.
Dal medesimo smartphone, utilizzando il browser, riesco ad accedere senza problemi.
Però il mancato accesso all'app impedisce di utilizzare le automazioni ecc. legate allo smartphone.
Non ho cambiato nulla nella configurazione: con l'altro smartphone difatti riesco ad accedere senza problemi.
Grazie a chiunque mi potrà aiutare.

**Comment by snakuzzo** [comment]
Ciao, con questo che hai scritto ci fai poco.
Da browser riesci ad accedere? Se sì, come ?
I log su HassOS (Supervisor + HA) li hai visti ?

**Reply by amadeus74** [reply]
Ciao.
Da browser riesco, sia da PC che dallo smartphone su cui l'app sembra non funzionare.
Ho verificato i LOG di supervisor e core ma nel momento in cui cerco di accedere tramite app e sullo smartphone mi esce il messaggio riportato sopra, non esce alcuna nuova riga che segnali problemi o altro...
Se può servire, il telefono ha Android 7 e tutte le app e i servizi aggiornati.

**Reply by snakuzzo** [reply]
come hai configurato l'app?  Che URL hai impostato ?

**Reply by amadeus74** [reply]
Ecco, è prioprio questo il problema: non ho più da nessuna parte la possibilità di configurare le impostazioni dell'app...
Se sono sulla stessa rete wifi di HASSIO mi propone l'istanza, altrimenti inserisco manualmente https://nomeistanza.duckdns.org:8123
Mi chiede l'account per accedere e poi va in errore.
Invece da browser entro senza problemi.

**Reply by snakuzzo** [reply]
devi guardare i log per capire cosa accade quando provi ad accedere da app...accedi dal PC e vai a guardare i log di home assistant

**Reply by amadeus74** [reply]
Ho provato a guardare tutti i LOG, ma non compare nulla quando provo ad accedere da app.
Non compare anche quando accedo da appa da un altro smartphone...

**Reply by snakuzzo** [reply]
Se non appare nulla vuol dire che non ci sono errori. Quindi mi verrebbe da dire che dall'app che hai sullo smartphone con android 7, nemmeno ci arrivi a contattare HA.
Prova da unaltro smartphone.
Comunque cerca qui sulle discussioni perché se non ricordo male c'era stato già un caso simile

**Reply by amadeus74** [reply]
Ma fino a un po' di tempo fa funzionava senza problemi, e il telefono ha sempre avuto Android 7!
Temo anch'io che sia un problema dell'app, ma anche cancellando la cache e i dati non cambia nulla...
Cercherò nel forum per vedere se trovo qualcosa.
Grazie per l'aiuto!
